src/main/java/com/redhat/pantheon/use/ModuleData.java

```diff
@@ -8,40 +8,51 @@
 
 import javax.jcr.query.Query;
 import javax.script.Bindings;
+
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import javax.inject.Inject;
+
 
 /**
  * Created by ben on 3/5/19.
  */
 public class ModuleData implements Use {
 
-    private ResourceResolver resolver;
+    private static ResourceResolver resolver;
```

Comment:
Not sure making this variable `static` is a good idea here. If multiple `ModuleData` objects are used simultaneously (which is likely the case), sling will just overwrite the resolver every time.

Reply:
I think my understanding of the resolver is was mistaken, I was thinking that I did not care if it got replaced because I would get the new state of the data instead. And given that I mean just looking to see what modules are available It does not matter if the data was updated just as I was searching.

Reply:
However, I have updated the code with Ben's suggestions and this takes care of my initial need for a static resolver.